examples/error-lifting/error_lifting.description

```diff
@@ -1,6 +1,3 @@
 // The error lifting operator enables you to navigate through a set of fields without worrying about 
-// error or nil values along the way. In an event of an error or a nil return, the operator stops
+// `error` or `nil` values along the way. In the event of an `error` or a `nil` return, the operator stops
```

Comment:
```suggestion
// `error` or `nil` values along the way. In the event of an `error` or a `nil` being returned, the operator stops
```

Reply:
The current version is correct right? Since it is "In the event of an ..."

Reply:
the event itself is the error or nil return right? Do we need the comma?

Reply:
Let's say, "if an `error` or a `nil` is returned,